packages/ui/src/ui/components/JobStatisticTable/JobStatisticTable.tsx

```diff
@@ -0,0 +1,180 @@
+import * as React from 'react';
+import {useCallback} from 'react';
+import cn from 'bem-cn-lite';
+
+import ErrorBoundary from '../ErrorBoundary/ErrorBoundary';
+import ElementsTableRaw from '../ElementsTable/ElementsTable';
+import Icon from '../Icon/Icon';
+import Toolbar from './Toolbar';
+
+import {JobTreeItem, LeafStatistic} from '../../types/job';
```

Comment:
It would be better to avoid usage of Job-specific types. It looks like yql_statistics has the same format in response like job-statistics, so it would be better to handle raw-format in the component. We can React.useReducer and extract all necessary logic from corresponding reducers/actions/selectors.